src/regression.js

```diff
@@ -18,6 +18,32 @@ module.exports = {
     console.log('yVals are:', yVals);
     console.log('localInitialMVals are:', localInitialMVals);
     /* eslint-enable no-console */
-    return n.uncmin(w => this.objective(w, xVals, yVals), localInitialMVals, 0.0001).solution;
+    return n.uncmin(w => this.objective(w, xVals, yVals), localInitialMVals, 0.001).solution;
   },
+
+  rSquared(xVals, yVals, betaVector) {
```

Comment:
Some comments on this function:

* I’d like to see some parameter documentation, which should eventually help us abstract this out
* Please add unit tests
* What does `SSresidual` and `SStotal` stand for? These variable names don’t add meaning to their values
* We’re computing averages all over the place. Might be nice to create a helper function.